k8s/helm-chart/kviklet.plain-secret.yaml

```diff
@@ -0,0 +1,18 @@
+# This a example plain secret please encrypt it self 
+apiVersion: v1
+stringData:
+    values.yaml: |
+        config:
+          datasource:
+            username: appkviklet
+            password: databasepassword
+          initialUser:
+            email: admin
```

Comment:
maybe make this an actual email? I know the tool currently doesn't require this but if we end up adding email based features it would be helpful if every user actually has one :D

Reply:
Ok let me update later